client/signup/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -13,120 +13,161 @@ import { urlToSlug } from 'lib/url';
+			status: 'is-error',
+		},
+
+		notActiveJetpack: {
+			icon: 'block',
+			text: this.props.translate( 'Jetpack is deactivated.' ),
+			status: 'is-error',
+		},
+
+		outdatedJetpack: {
+			icon: 'block',
+			text: this.props.translate( 'You must update Jetpack before connecting.' ),
+			status: 'is-error',
+		},
+
+		jetpackIsDisconnected: {
+			icon: 'link-break',
+			text: this.props.translate( 'Jetpack is currently disconnected.' ),
+			status: 'is-error',
+		},
+
+		jetpackIsValid: {
+			status: 'is-success',
+			icon: 'plugins',
+			text: this.props.translate( 'Jetpack is connected.' ),
+		},
+
+		notJetpack: {
+			status: 'is-notice',
+			icon: 'status',
+			text: this.props.translate( 'Jetpack couldn\'t be found.' ),
+		},
+
+		// same as alreadyOwned
+		alreadyConnected: {
+			status: 'is-success',
+			icon: 'status',
+			text: this.props.translate( 'This site is already connected!' ),
+			showDismiss: false,
+			children: (
 				<NoticeAction href={ '/plans/my-plan/' + urlToSlug( this.props.url ) }>
-					{ translate( 'Visit' ) }
+				{ this.props.translate( 'Visit' ) }
 				</NoticeAction>
-			);
-			return noticeValues;
-		}
-		if ( this.props.noticeType === 'wordpress.com' ) {
-			noticeValues.text = translate( 'Oops, that\'s us.' );
-			noticeValues.status = 'is-warning';
-			noticeValues.icon = 'status';
-			return noticeValues;
-		}
-		if ( this.props.noticeType === 'retryingAuth' ) {
-			noticeValues.text = translate( 'Error authorizing. Page is refreshing for an other attempt.' );
-			noticeValues.status = 'is-warning';
-			noticeValues.icon = 'notice';
-			return noticeValues;
-		}
-		if ( this.props.noticeType === 'retryAuth' ) {
-			noticeValues.text = translate( 'In some cases, authorization can take a few attempts. Please try again.' );
-			noticeValues.status = 'is-warning';
-			noticeValues.icon = 'notice';
-			return noticeValues;
-		}
-		if ( this.props.noticeType === 'secretExpired' ) {
-			noticeValues.text = translate( 'Oops, that took a while. You\'ll have to try again.' );
-			noticeValues.status = 'is-error';
-			noticeValues.icon = 'notice';
-			return noticeValues;
-		}
-		if ( this.props.noticeType === 'defaultAuthorizeError' ) {
-			noticeValues.text = translate( 'Error authorizing your site. Please {{link}}contact support{{/link}}.', {
+			),
+		},
+
+		// same as alreadyConnected
+		alreadyOwned: {
+			status: 'is-success',
+			icon: 'status',
+			text: this.props.translate( 'This site is already connected!' ),
+			showDismiss: false,
+			children: (
+				<NoticeAction href={ '/plans/my-plan/' + urlToSlug( this.props.url ) }>
+				{ this.props.translate( 'Visit' ) }
+				</NoticeAction>
+			),
+		},
+
+		'wordpress.com': {
+			text: this.props.translate( 'Oops, that\'s us.' ),
+			status: 'is-warning',
+			icon: 'status',
+		},
+
+		retryingAuth: {
+			text: this.props.translate( 'Error authorizing. Page is refreshing for another attempt.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Error authorizing. Page is refreshing for an other attempt.' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).